packages/blocks/src/root-block/widgets/format-bar/config.ts

```diff
@@ -317,21 +322,55 @@ export function toolbarDefaultConfig(toolbar: AffineFormatBarWidget) {
     });
 }
 
-export function toolbarMoreButton(toolbar: AffineFormatBarWidget) {
-  const actions = [
-    [
+export class FormatBarContext extends MoreMenuContext {
+  constructor(public toolbar: AffineFormatBarWidget) {
+    super();
+  }
+
+  override isEmpty() {
+    return this.selectedBlockModels.length === 0;
+  }
+
+  get doc() {
+    return this.toolbar.host.doc;
+  }
+
+  get host() {
+    return this.toolbar.host;
+  }
+
+  get selectedBlockModels() {
+    const [result, { selectedModels }] = this.std.command
```

Comment:
Can be `const { result, selectedModels } = this.std.command.exec('getSelectedModels')`;